### PR TITLE
Simple Email System Functioning

### PR DESCRIPTION
**Pull request by Whomstt**
Run NotificationServiceApplication.java to send an email from lastmanstanding.notifier@gmail.com to itself / change recipient email in NotificationServiceApplication.java if you want to send it to another email for testing purposes.

### CODE REVIEW

File: notification-service/.gitattributes
Comment (olanhealy) [inline notification-service/.gitattributes:1]: Not sure if needed

File: notification-service/.mvn/wrapper/maven-wrapper.properties
Comment (olanhealy) [inline notification-service/.mvn/wrapper/maven-wrapper.properties:1]: Not sure again if needed

File: notification-service/mvnw
Comment (olanhealy) [inline notification-service/mvnw:1]: again not sure if needed

File: notification-service/mvnw.cmd
Comment (olanhealy) [inline notification-service/mvnw.cmd:1]: Not sure if needed

File: notification-service/src/main/resources/application.properties
Comment (olanhealy) [inline notification-service/src/main/resources/application.properties:1]: Dont think this info should have been committed? Maybe should just be in a .env for each of us we don't commit?

File: notification-service/src/test/java/com/lms/notificationservice/NotificationServiceApplicationTests.java
Comment (olanhealy) [inline notification-service/src/test/java/com/lms/notificationservice/NotificationServiceApplicationTests.java:1]: May need to comment out in future something mvn clean install will not pass when you have methods written up and have test like this

**Review by olanhealy — CHANGES_REQUESTED** [review]
Very good for most part and is described as to design doc. Good work.
Just left small comments:
- Not sure some files are needed
- The application properties maybe shouldn't have that info on github

File: notification-service/src/main/resources/application.properties
Comment (Whomstt) [reply notification-service/src/main/resources/application.properties:1]: You're right. Deleted old app password and generated a new one that won't be committed. Now stored in env file.

File: notification-service/src/test/java/com/lms/notificationservice/NotificationServiceApplicationTests.java
Comment (Whomstt) [reply notification-service/src/test/java/com/lms/notificationservice/NotificationServiceApplicationTests.java:1]: Commented out now

**Review by olanhealy — APPROVED** [review]
LGTM, good work

**Comment by CaoimheCahill** [comment]
Looks good to me

**Review by CaoimheCahill — APPROVED** [review]

**Pull request merged**